client/src/components/AboutCard/__snapshots__/AboutCard.test.tsx.snap

```diff
@@ -0,0 +1,56 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`rendering of the AboutCard checks if component renders 1`] = `
+<DocumentFragment>
+  <div
+    class="grid-gap-lg tablet:grid-col"
```

Comment:
The advantage of using global classname is that these are defined in jest

Reply:
Oh good point!